README.md

```diff
@@ -71,13 +71,26 @@ For currently supported operations, see [Commands](##Commands) section.
 
 ### From npm
 
+To install globally:
+
+```sh
+npm install --global github:ethersphere/swarm-cli
```

Comment:
This would install from the github repo from source from master branch, not from npmjs.com where the latest released versions are stored.

```suggestion
npm install --global @ethersphere/swarm-cli
```

Reply:
oh, i didn't know, sorry! updated the PR.